tower-fallback/src/lib.rs

```diff
@@ -8,7 +8,7 @@
 //!
 //! TODO: compare with similar code in linkerd.
 //!
-//! [aws-fallback]: https://aws.amazon.com/builders-library/avoiding-fallback-in-distributed-systems/
+//! [aws-fallback]: <https://aws.amazon.com/builders-library/avoiding-fallback-in-distributed-systems/>
```

Comment:
These are interesting because they should just link in the reference citing them and not render the full url itself, is rustdoc complaining about this?

Reply:
I did more investigation, and rustdoc actually doesn't complain here. This is what confused me:

![image](https://user-images.githubusercontent.com/8863307/170492922-7e860bb9-60f9-48a7-b74c-2d996c347de4.png)

When I saw this error, I thought such links have to be enclosed in `<` `>` as well. Here's the comments from the code: 
```rust
    /// Check if the `network_upgrade` fields from each transaction in the block matches
    /// the network upgrade calculated from the `network` and block height.
    ///
    /// # Consensus
    ///
    /// > [NU5 onward] The nConsensusBranchId field MUST match the consensus branch ID used
    /// > for SIGHASH transaction hashes, as specified in [ZIP-244].
    ///
    /// <https://zips.z.cash/protocol/protocol.pdf#txnconsensus>
    /// [ZIP-244]: https://zips.z.cash/zip-0244
```
The problem was that there wasn't a newline between the two links at the bottom. 

I'm going to revert the reversion of the reversion.

Reply:
Done in [a26e079](https://github.com/ZcashFoundation/zebra/pull/4494/commits/a26e0797b7fe5fd3ba2fbb50f4550daed654d918).